File: dataset/output/OPENAI_gpt-4o/Task62_OPENAI_gpt-4o.c
#include <stdio.h>
#include <json-c/json.h>

int main() {
    const char *jsonString = "{\"name\":\"John\", \"age\":30, \"car\":null}";
    struct json_object *parsed_json;
    struct lh_table *object;
    parsed_json = json_tokener_parse(jsonString);
    
    object = json_object_get_object(parsed_json);
    printf("Root Element: %s
", (char *)lh_table_get_key(object->head));
    
    return 0;
}

File: dataset/output/OPENAI_gpt-4o/Task62_OPENAI_gpt-4o.java
import org.json.JSONObject;

public class Main {
    public static void main(String[] args) {
        String jsonString = "{\"name\":\"John\", \"age\":30, \"car\":null}";
        JSONObject obj = new JSONObject(jsonString);
        System.out.println("Root Element: " + obj.keySet().toArray()[0]);
    }
}